CMakeLists.txt

```diff
@@ -171,7 +171,14 @@ ENDIF()
 SET(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}-${CPACK_RPM_PACKAGE_VERSION}_${version}-${stagever}${CPACK_SYSTEM_NAME}")
 MESSAGE ("-- Current release version is ${CPACK_PACKAGE_FILE_NAME}")
 
-find_file(CLIENTTOOLS_PACKAGE_FILE "hpccsystems-clienttools-community_${version}-${stagever}${CPACK_SYSTEM_NAME}.exe" HINTS ${CMAKE_CURRENT_BINARY_DIR}/../HPCC-Platform )
+IF(NOT "${CLIENTTOOLS_ARCH}" STREQUAL "")
+    SET(CLIENTTOOLS_CPACK_SYSTEM_NAME  "${CMAKE_SYSTEM_NAME}-${CLIENTTOOLS_ARCH}")
+ELSE()
+    SET(CLIENTTOOLS_CPACK_SYSTEM_NAME  "${CPACK_SYSTEM_NAME}")
+ENDIF()
```

Comment:
I don't think there is any point in making this a user settable thing, you could simply hard code the 64bit text?